src/atsamd/fdcan.c

```diff
@@ -234,12 +234,14 @@ compute_btr(uint32_t pclock, uint32_t bitrate)
 void
 can_init(void)
 {
+#if CONFIG_HAVE_SAMD_USB
     if (!CONFIG_USB) {
         // The FDCAN peripheral only seems to run if at least one
         // other peripheral is also enabled.
         enable_pclock(USB_GCLK_ID, ID_USB);
         USB->DEVICE.CTRLA.reg = USB_CTRLA_ENABLE;
     }
+#endif
```

Comment:
It's only necessary to enable one other device - if ID_USB isn't defined, we can choose some other device.  (This whole thing is some weird quirk of the samd power enable code on sam5x.)

Reply:
Do you know if this is true for the samc21? This chip does not have usb, so we’d have to enable something else.

Reply:
@KevinOConnor can you provide more details about this? I’m having issues with CAN, and am wondering where I might investigate this further for the samc21.